CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Additions and Improvements 
 
 - New`trace_replayBlockTransactions` JSON-RPC API
+- Ability to automatically detect Docker NAT settings from inside the conainter.
```

Comment:
I think this probably needs to go below the following paragraph -> the following paragraph describes the bullet point above this one. 

Is it enabled using a CLI option?  It'd be good to give the changelog reader some idea of how to enable this.